Yoast/wordpress-seo
# Introduce post meta watcher for BC with other plugins updating our meta values

## Summary


This PR can be summarized in the following changelog entry:

* Watches changes to postmeta outside of post saves to preserve BC with third parties updating our postmeta.

## Relevant technical choices:

* Uses a shutdown function so indexables are only saved once even if someone updates several meta one after the other.

## Test instructions

This PR can be tested by following these steps:

* Call `update_post_meta` anywhere that's not part of the usual save to edit one of our meta values.
* The value should be updated in our indexable as well.

## UI changes

## Documentation

## Quality assurance
